src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -1,14 +1,368 @@
 <template>
-<div></div>
+  <v-card flat id="share-structure">
+
+    <!-- Summary Section -->
+    <div id="share-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="share-summary-header" >
+        <v-icon>mdi-file-tree</v-icon>
+        <label class="share-summary-header-title"><strong> Share Structure</strong></label>
+      </div>
+
+      <!-- Summary Warning -->
+      <div v-if="showErrorSummary" class="share-summary-invalid-message">
+        <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
+          This step is not complete.
+          <router-link id="router-link" :to="{ path: '/create-share-structure', query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <v-data-table
+      :headers="headers"
+      :items="shareClasses"
+      :disable-pagination="true"
+      :disable-sort="true"
+      :hide-default-footer="true"
```

Comment:
I think these props have a shortcut: you can just specify their name with no value, eg
```
<v-data-table
   disable-pagination
   disable-sort
   ...
```

Reply:
Fixed